recipes/openvino/all/conanfile.py

```diff
@@ -226,6 +228,7 @@ def generate(self):
         toolchain.cache_variables["ENABLE_OV_TF_LITE_FRONTEND"] = self.options.enable_tf_lite_frontend
         toolchain.cache_variables["ENABLE_OV_ONNX_FRONTEND"] = self.options.enable_onnx_frontend
         toolchain.cache_variables["ENABLE_OV_PYTORCH_FRONTEND"] = self.options.enable_pytorch_frontend
+        toolchain.cache_variables["ENABLE_OV_JAX_FRONTEND"] = self.options.enable_jax_frontend
```

Comment:
```suggestion
        if Version(self.version) >=  "2024.3.0":
            toolchain.cache_variables["ENABLE_OV_JAX_FRONTEND"] = self.options.enable_jax_frontend
```

This options does not exist before 2024.3.0: https://github.com/openvinotoolkit/openvino/compare/2024.2.0...2024.3.0#diff-dd9d470bfd7d5c3c875bd6f1c182ac653ef5fb312ae28c24b8ea39c5074e7c8e